webpack.config.js

```diff
@@ -21,11 +21,13 @@ const availableStagingLocales = process.env.L10N
   : JSON.stringify(undefined);
 
 const config = {
-  entry: ['./src/index'],
+  entry: {
+    main: './src/index',
+    serviceWorkerCompat: './src/service-worker/offline-plugin-compat',
+  },
   output: {
     path: path.join(__dirname, 'dist'),
-    filename: '[hash].bundle.js',
-    chunkFilename: '[id].[hash].bundle.js',
```

Comment:
I noticed that the default for `chunkFilename` gives the same result so I removed it as well, but I can add it back if you prefer.

Reply:
I don't have a strong preference. I think it's fine to remove if this is the default already.